backend/protocol_rpc/endpoints.py

```
@@ -241,16 +244,18 @@ async def create_random_validator(
 
 @check_forbidden_method_in_hosted_studio
 async def create_random_validators(
-    validators_registry: ModifiableValidatorsRegistry,
-    accounts_manager: AccountsManager,
-    llm_provider_registry: LLMProviderRegistry,
+    session: Session,
     validators_manager: validators.Manager,
     count: int,
     min_stake: int,
     max_stake: int,
     limit_providers: list[str] = None,
```

Ruff (0.13.1)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)